messages/CMakeLists.txt

```diff
@@ -55,7 +46,11 @@ add_custom_command(
   OUTPUT ${OUTPUT_SOURCES} ${OUTPUT_HEADERS}
   DEPENDS ${PROTO_FILES} ${PROTO_OPTION_FILES}
   COMMAND
-  ${PROTOC} "--proto_path=${CMAKE_CURRENT_SOURCE_DIR}" "--nanopb_out=-I${CMAKE_CURRENT_SOURCE_DIR}:${CMAKE_CURRENT_BINARY_DIR}" ${PROTO_FILES_ABSOLUTE}
+  ${CMAKE_SOURCE_DIR}/tools/nanopb/generator/protoc
```

Comment:
Since you here, would it make sense to move nanopb to `/external` dir? For comparison, FIDO2's tinycbor is also in external.

Reply:
can you do it after this is merged? submodules are my weak point :innocent:

Reply:
ah, yes course!